themes/developers/lib/functions/load-assets.php

```diff
@@ -5,32 +5,50 @@
  * @package     spiralWebDB\Developers
  * @since       1.0.0
  * @author      Robert A. Gadon
- * @link        https://knowthecode.io
+ * @link        https://spiralwebdb.com
  * @license     GNU General Public License 2.0+
  */
 
 namespace spiralWebDB\Developers;
 
+add_filter( 'stylesheet_uri', __NAMESPACE__ . '\change_stylesheet_uri_to_min' );
+/**
+ * Change the stylesheet to the minified version.
+ *
+ * @since 2.0.0
+ *
+ * @param string $stylesheet_uri Stylesheet's URI.
+ *
+ * @return string
+ */
+function change_stylesheet_uri_to_min( $stylesheet_uri ) {
+	if ( is_in_debug() ) {
+		return $stylesheet_uri;
+	}
+
+	return get_theme_url() . '/style.min.css';
+}
```

Comment:
@rgadon107 When WordPress sets the stylesheet's URI, this code hooks into its filter to change it from the theme's `style.css` file to the minified/optimized version, ie `style.min.css`.